### PR TITLE
Update mxnet eia notebook examples

### PR DESCRIPTION
**Pull request by ycnie**
*Issue #, if available:*

*Description of changes:*
- Update mxnet eia notebook examples:
  - `mxnet_mnist_elastic_inference.ipynb`
  - `mxnet_mnist_elastic_inference_local.ipynb`

- Added script `mxnet_ei.py`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mchoi8739** [comment]
1. Would you confirm if this runs end-to-end on your NI and verify that the endpoint and predictor works as expected?
2. There's an undefined object, `data`. Can you split the dataset into train, validation, and test datasets, and properly define the `data` object for the prediction part? Related issue: https://github.com/aws/amazon-sagemaker-examples/issues/2033

```

* mxnet_mnist_elastic_inference.ipynb ******************************************

*

*    job name: papermill-mxnet-mnist-elastic-inference-2021-02-25-15-01-03

*

*      status: Failed     

*



---------------------------------------------------------------------------

Exception encountered at "In [7]":

---------------------------------------------------------------------------

NameError                                 Traceback (most recent call last)

<timed exec> in <module>



NameError: name 'data' is not defined

```

**Comment by ycnie** [comment]
It worked fine on my end with SageMaker Python SDK version `2.26.0`. The `data` object I believe comes from the existing code where takes the HTML input as the instructions in this `Making an inference request` section shows: 
> To see inference in action, draw a digit in the image box below. The pixel data from your drawing is loaded into a variable named data.
```python
from IPython.display import HTML
HTML(open("input.html").read())
```
Is that the reason why the `data` was undefined in the test?

**Comment by ycnie** [comment]
@hsl89 Thanks for confirming. Sure I can do other tests but it doesn't add evidence to prove the existing code above works fine in addition to running this notebook in a SageMaker Notebook Instance on my end.
I think it's an existing issue that automated test has with this notebook as is shown in this [Bug Report](https://github.com/aws/amazon-sagemaker-examples/issues/2033). Is there any plan for the fixing?

**Comment by hsl89** [comment]
Hi @ycnie, we definitely have plan to make a more robust testing system. But the interactive inputs like  
```
from IPython.display import HTML
HTML(open("input.html").read())
``` 
will still not be recommended

File: sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_elastic_inference.ipynb [hunk 1]
```diff
@@ -56,7 +56,7 @@
    "source": [
```
Comment (hsl89) [inline sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_elastic_inference.ipynb:56]: <p>use for loop can make this cell shorter </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_elastic_inference_local.ipynb [hunk 2]
```diff
@@ -80,7 +80,7 @@
    "source": [
```
Comment (hsl89) [inline sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_elastic_inference_local.ipynb:80]: <p>replace this part by some sample data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Pull request merged**